crates/task-impls/src/quorum_vote.rs

```diff
@@ -334,7 +336,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumVoteTaskState<TYPES, I
             );
 
             // Cancel the old dependency tasks.
-            for view in (*self.latest_voted_view + 1)..=(*new_view) {
+            for view in (*self.latest_voted_view)..=(*new_view) {
```

Comment:
Do we need to include `self.latest_voted_view` here? E.g., if `self.latest_voted_view` is 3, then previously, the `new_view` is 3, so we've already canceled the task for view 3. Now we only have to cancel the tasks from view 4.

Reply:
Yeah I was trying to debug if I had any long-running tasks laying around. I've reverted this